packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css

```diff
@@ -195,3 +195,8 @@ html:not([data-theme='dark']) .redocusaurus div[id^='react-tabs'] code {
 .redocusaurus table.security-details tr:nth-child(2n + 1) {
   background-color: var(--ifm-background-surface-color);
 }
+
+/* Authorisation Dropdown background color fix */
+[data-theme='dark'] .redocusaurus .api-content .sc-jQHtVU:has(span > span > code) {
```

Comment:
`.sc-jQHtVU` class name will change whenever there is a new build of redoc available so I don't think this fix should be merged. There should be a better way to do this